### PR TITLE
webView: Create util function to identify whether target is reaction.

### PR DESCRIPTION
**Pull request by jainkuniya**
And pass it on long press event message.

part 1 of #2252 

(Let's implement #2252 in smaller part, in several smaller commits and PR's. Easy for review and maintain 😃 )

### CODE REVIEW

File: src/render-html/js.js [hunk 1]
```diff
@@ -51,6 +51,9 @@ const getMessageNode = (node: Element): Element => {
   return curNode;
 };
 
+const isTargetIsReaction = (node: Element): boolean =>
```
Comment (borisyankov) [inline src/render-html/js.js]: `isTargetIsReaction` => `isTargetReaction`

File: src/render-html/js.js [hunk 2]
```diff
@@ -204,7 +207,11 @@ const handleLongPress = e => {
 
   sendMessage({
     type: 'longPress',
-    target: e.target.matches('.header') ? 'header' : 'message',
+    target: e.target.matches('.header')
```
Comment (borisyankov) [inline src/render-html/js.js]: Are we handling a thing pressed is empty space?
Not related to this PR but the code made me thing...

File: src/render-html/js.js [hunk 3]
```diff
@@ -204,7 +207,11 @@ const handleLongPress = e => {
...
+      ? 'header'
+      : isTargetIsReaction(e.target)
+        ? 'reaction-list'
+        : 'message',
```
Comment (borisyankov) [inline src/render-html/js.js]: This code is asking to be a separate function at this point.

File: src/render-html/js.js (hunk 2 above)
Comment (borisyankov) [reply src/render-html/js.js]: Also, I am wondering if we should use the `node.classList.contains('header')` call?

File: src/render-html/js.js (hunk 1 above)
Comment (jainkuniya) [reply src/render-html/js.js]: yup!

**Comment by gnprice** [comment]
There's other work more recently on other PRs attached to this issue, but I think this PR itself is no longer relevant.

**Pull request closed without merging**